content/applications/productivity/studio/views.rst

```diff
@@ -0,0 +1,377 @@
+
+   .. image:: views/kanban-project.png
+      :align: center
+      :alt: Project model's Kanban view
+
+.. _studio/views/multiple-records/list:
+
+List
+----
+
+The :guilabel:`List` view is used to overview many records at once, look for records, and edit
+simple records.
+
+- To prevent users from creating, editing, or deleting records, untick :guilabel:`Can Create`,
+  :guilabel:`Can Edit`, or :guilabel:`Can Delete`.
+- To create and edit records directly within the view, select either :guilabel:`New record on top`
+  or :guilabel:`New record at the bottom` under :guilabel:`Editable`.
+
+  .. note::
+     This prevents users from opening records in :ref:`Form view <studio/views/general/form>` from the
+     :guilabel:`List` view.
+
+- To edit several records at once, tick :guilabel:`Enable Mass Editing`.
+- To change the way records are sorted by default, select a field under :guilabel:`Sort By`.
+
+.. tip::
+   To add a drag handle icon to reorder records manually, add an :ref:`Integer field
+   <studio/fields/simple-fields/integer>` with the :guilabel:`Handle` widget.
+
+   .. image:: views/list-drag-handle.png
+      :align: center
+      :alt: Drag handle icon enabling to sort records manually in List view
+
+.. example::
+
+   .. image:: views/list-sales-order.png
+      :align: center
+      :alt: Sales order model's List view
+
+.. _studio/views/multiple-records/map:
+
+Map
+---
+
+The :guilabel:`Map` view is used to display records on a map. For example, it is used in the Field
+Service app to plan an itinerary between different tasks.
+
+.. note::
+   A :ref:`Many2One field <studio/fields/relational-fields/many2one>` linked to the *Contact* model is required to
+   activate the view, as the contact address is used to position records on the map.
+
+- To select which kind of contact should be used on the map, select it under :guilabel:`Contact
+  Field`.
+- To hide the name or the address of the record, tick :guilabel:`Hide name` or :guilabel:`Hide
+  Address`.
+- To add information from other fields, select them under :guilabel:`Additional Fields`.
+- To have a route suggested between the different records, tick :guilabel:`Enable Routing` and
+  select which field should be used to sort records for the routing.
+
+.. example::
+
+   .. image:: views/map-task.png
+      :align: center
+      :alt: Task model's Map view
+
+.. _studio/views/timeline:
+
+Timeline views
+==============
+
+.. note::
+   When you first activate one of the timeline views, you need to select which :ref:`Date
+   <studio/fields/simple-fields/date>` or :ref:`Date & Time <studio/fields/simple-fields/date-time>` fields on your model should
+   be used to define when the records start and stop in order to display them on the view. You can
+   modify the :guilabel:`Start Date Field` and :guilabel:`Stop Date Field` after activating the
+   view.
+
+   The settings described below are found under the view's :guilabel:`View` tab unless specified
+   otherwise.
```

Comment:
By "combine" I mean "in a bulleted list" but both look fine IMO.
```suggestion
   - When you first activate one of the timeline views, you need to select which :ref:`Date
   <studio/fields/simple-fields/date>` or :ref:`Date & Time <studio/fields/simple-fields/date-time>` fields on your model should
   be used to define when the records start and stop in order to display them on the view. You can
   modify the :guilabel:`Start Date Field` and :guilabel:`Stop Date Field` after activating the
   view.
   - The settings described below are found under the view's :guilabel:`View` tab unless specified
   otherwise.
```